packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -264,13 +222,13 @@ function Session(props: PropsWithChildren<Props>) {
     tdpClient,
     username,
     hostname,
-    clipboardSharingEnabled,
     setClipboardSharingEnabled,
```

Comment:
I'm not sure this is used any longer?

Reply:
https://github.com/gravitational/webapps/pull/1333#discussion_r1022069855